customer/src/test/java/no/unit/nva/customer/update/UpdateControlledVocabularyHandlerTest.java

```diff
@@ -94,20 +95,25 @@ public void handleRequestReturnsUnsupportedTypeWhenAcceptedContentTypeIsNotSuppo
     }
 
     @Test
-    public void handleRequestReturnsConflictWhenCustomerAlreadyHasVocabularySettings()
-        throws IOException, NotFoundException {
-        CustomerDto customerWithoutVocabularySettings = CustomerDataGenerator
-            .createSampleCustomerDto().copy()
-            .withVocabularies(Collections.emptySet())
-            .build();
-        customerService.createCustomer(customerWithoutVocabularySettings);
-        var response = sendRequestAcceptingJsonLd(customerWithoutVocabularySettings.getIdentifier()).getResponse();
+    public void handleRequestReturnsBadRequestWhenCustomerAlreadyHasVocabularySettingsAndWeAttemptToEraseThem()
```

Comment:
had to update the test because it was failing and the implementation did not make much sense.